WcaOnRails/app/models/light_result.rb

```diff
@@ -0,0 +1,107 @@
+require 'solve_time'
+require_relative './format'
+
+class LightResult
```

Comment:
You have a very nice description of why this class exists here: https://github.com/timhabermaas/worldcubeassociation.org/commit/511ac52955bcdeac5fcb3095e9e3afbd6a6eec7e. Could you add that description to this class?